ci.yml

```diff
@@ -47,19 +53,24 @@ stages:
             displayName: 'Build and Package'
             inputs:
               mavenPomFile: 'pom.xml'
-              goals: 'package'
-              options: '$(DefaultOptions) $(ProfileFlag) "-DpackageOutputDirectory=$(Build.ArtifactStagingDirectory)" -Dmaven.test.skip=true'
+              goals: 'deploy'
```

Comment:
I think we need another set of double colons. 😆

Reply:
I'm voting for the introduction of the delimiter delimiter delimiter character in unicode.